sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisErrorSuite.scala

```diff
@@ -457,7 +457,7 @@ class AnalysisErrorSuite extends AnalysisTest {
             AttributeReference("b", IntegerType)(exprId = ExprId(1))))
 
       if (shouldSuccess) {
-        assertAnalysisSuccess(plan, true)
+        assertAnalysisSuccess(plan)
```

Comment:
I wouldn't remove these even if it's the param default value. It might be a little clearer to read explicitly, and won't change if the default changes (though it almost surely won't)

Reply:
+1 with @srowen . It is not helpful to remove these parameters.

Reply:
@srowen @gengliangwang Got it. I just reverted this change.